tools/fix-h1s.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+# Assume Dokka has been run
+pushd "`dirname "$0"`"/../docs/html
+
+# Make the output SEO friendly by converting the "h2" title to the proper "h1".
+# For Dokka, this is only an issue on the index page (apparently).
+sed -i -e 's|<h2\(.*\)</h2>|<h1\1</h1>|' index.html
```

Comment:
Goal is to change this h2 to an h1.
<img width="1400" alt="image" src="https://github.com/realm/realm-kotlin/assets/20050130/de823dad-5123-452a-9cbb-7206619f34e1">

Reply:
I have verified the updated script by running 
```
../tools/fix-h1s.sh build/dokka/htmlMultiModule
``` 
inside `packages`. 

It now yields the desired `h1` headings:
<img width="696" alt="Screenshot 2023-07-25 at 10 41 48" src="https://github.com/realm/realm-kotlin/assets/12733934/358909c6-f9f5-44c4-b29b-721ffe423ee3">